### PR TITLE
Adding info #5247

### PR DESCRIPTION
**Pull request by j0rt3g4**
Adding information for issue #5247 
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: windows/privacy/configure-windows-diagnostic-data-in-your-organization.md [hunk 1]
```diff
@@ -142,6 +142,11 @@ The data transmitted at the Basic and Enhanced data diagnostic levels is quite s
 
 The Microsoft Data Management Service routes data back to our secure cloud storage. Only Microsoft personnel with a valid business justification are permitted access.
 
+Solutions like Desktop Analytics or Microsoft Defender Advanced Threat Protection need Windows devices to reach diagnostics endpoints to enable organizations leverage solutions based on diagnostics data. These solutions leverage Windows components like the Connected User Experiences and Telemetry service, Windows Defender Advanced Threat Protection service, Windows Error Reporting, and Online Crash Analysis.
+
+For a complete list of diagnostics endpoints leveraged by Desktop Analytics, see [here](https://docs.microsoft.com/configmgr/desktop-analytics/enable-data-sharing).
```
Comment (JohanFreelancer9) [inline windows/privacy/configure-windows-diagnostic-data-in-your-organization.md]:
```suggestion
For a complete list of diagnostics endpoints leveraged by Desktop Analytics, see [Enable data sharing for Desktop Analytics](https://docs.microsoft.com/configmgr/desktop-analytics/enable-data-sharing).
```

File: windows/privacy/configure-windows-diagnostic-data-in-your-organization.md [hunk 2]
```diff
@@ -142,6 +142,11 @@ The data transmitted at the Basic and Enhanced data diagnostic levels is quite s
...
+For a complete list of diagnostics endpoints leveraged by Microsoft Defender Advanced Threat Protection, see [here](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet#enable-access-to-microsoft-defender-atp-service-urls-in-the-proxy-server).
```
Comment (JohanFreelancer9) [inline windows/privacy/configure-windows-diagnostic-data-in-your-organization.md]:
```suggestion
For a complete list of diagnostics endpoints leveraged by Microsoft Defender Advanced Threat Protection, see [Enable access to Microsoft Defender ATP service URLs in the proxy server](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet#enable-access-to-microsoft-defender-atp-service-urls-in-the-proxy-server).
```

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review based on issue #5247.  

Thank you.

**Comment by illfated** [comment]
#sign-off

File: windows/privacy/configure-windows-diagnostic-data-in-your-organization.md [hunk 3]
```diff
@@ -142,6 +142,11 @@ The data transmitted at the Basic and Enhanced data diagnostic levels is quite s
 
 The Microsoft Data Management Service routes data back to our secure cloud storage. Only Microsoft personnel with a valid business justification are permitted access.
 
+Solutions like Desktop Analytics or Microsoft Defender Advanced Threat Protection need Windows devices to reach diagnostics endpoints to enable organizations leverage solutions based on diagnostics data. These solutions leverage Windows components like the Connected User Experiences and Telemetry service, Windows Defender Advanced Threat Protection service, Windows Error Reporting, and Online Crash Analysis.
```
Comment (JohanFreelancer9) [inline windows/privacy/configure-windows-diagnostic-data-in-your-organization.md]:
```suggestion
Solutions like Desktop Analytics or Microsoft Defender Advanced Threat Protection need Windows devices to reach diagnostics endpoints which enable organizations to leverage solutions based on diagnostics data. These solutions leverage Windows components like the Connected User Experiences and Telemetry service, Windows Defender Advanced Threat Protection service, Windows Error Reporting, and Online Crash Analysis.
```

**Comment by mypil** [comment]
@j0rt3g4 -  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by j0rt3g4** [comment]
@mypil Done

**Comment by illfated** [comment]
#sign-off

**Pull request merged**